fix(postcss-svgo): skip non-SVG urls

ludofischer opened the pull request:
Fix #1050

alexander-akait commented on packages/postcss-svgo/src/__tests__/index.js:
```diff
@@ -226,4 +226,12 @@ test('should not crash on malformed urls when encoded', () => {
+test(
+  'should skip non-SVG urls',
+  passthroughCSS(`@font-face {
+  src: url("../example/dfds.woff2") format("woff2"),
```
Can we add test with relative SVG URL?

ludofischer commented:
Updated. I suspect this is another consequence of https://github.com/cssnano/cssnano/pull/1034 When we removed `is-svg` we started processing too many things that were not SVGs.

alexander-akait commented:
We can check it using `\.svg` extension, don't forget about `?something=bar` in query strings

internalsystemerror commented:
@ludofischer @alexander-akait thank you both
